README.md

```diff
@@ -1,24 +1,151 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [📹 video presentation](#video-presentation)
+  - [:camera: screenshots](#screenshots)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
 
-* Ruby version
+- [📝 License](#license)
 
-* System dependencies
+# 💰 Budget App <a name="about-project"></a>
 
-* Configuration
+> mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-* Database creation
+## 📹 video presentation <a name="Presentation"></a>
 
-* Database initialization
+[Video Presentation](https://www.loom.com/share/ba03ecea9b7c4e3197abeda47d615f8e)
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
+## 🛠 Built With <a name="built-with"></a>
 
-* Deployment instructions
+- <a href="https://www.ruby-lang.org/es/">ROR</a>
+- <a href="https://www.postgresql.org/">Postgresql</a>
+- <a href="https://rspec.info/">Rspec</a>
 
-* ...
+## 📹 Live Demo <a name="live-demo"></a>
+
+  [Live Link](https://budget-app-st2t.onrender.com)
```

Comment:
- [ ] Your live demo doesn't work as expected When I sign up I can't confirm my email which means I can log in and make a different transaction with your app. Please make sure that your app is deployed properly, the home and detail page works as expected. 👍 

![render](https://github.com/Chaseknis/budget_app/assets/6615394/5fe61433-d39b-4ba1-a6ab-f2e4418d0bc7)